config/api.yml

```diff
@@ -2518,9 +2518,12 @@
     :identifier: miq_report_reports
     :options:
     - :subcollection
-    :verbs: *gp
+    :verbs: *gpd
     :klass: MiqSchedule
     :subcollection_actions:
```

Comment:
maybe add the post equivalent (POST action "delete") for completeness.  No additional coded needed just the added specs.

Reply:
unless you wanted this PR for just the DELETE

Reply:
:+1: done